### ISSUE TITLE
Add term dorsiflexed posture of the feet

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10353]
**Preferred term label:**
Dorsiflexed posture of the feet

**Synonyms**
Dorsiflexion feet

**Definition (free text, please give PubMed ID)**
Involuntary dorsiflexion of the feet.

**Parent term (use hpo.jax.org/app)**
Abnormal posturing HP:0002533

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For example PMID37075751

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw @kingmanzhang 
Is this referring to a kind of contracture? How does this differ from the terms here:
https://hpo.jax.org/app/browse/term/HP:0034677

**Comment by pnrobinson** [comment]
@shahrzadnema -- adding this term, please see above
@azankl 
Adding new term: 
- ID: HP:6001097
- label: Dorsal flexion contracture of the ankle
- parents: [Ankle contracture (HP:0034677)](https://hpo.jax.org/app/browse/term/HP:0034677)
- pmid: 
- def: A type of contracture in which the dorsal flexion muscles of the ankle are contracted.
- comment: Dorsiflexion of the ankle is the movement of the top of the foot towards the leg in an upward motion. With dorsal flexion contracture, the ability to passively or actively move the tip of the toe away from the leg is limited.
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10353
- synonyms: Dorsiflexion contracture of the ankle(NONE)
